lib/modules/versioning/distro.spec.ts

```diff
@@ -87,16 +97,31 @@ describe('modules/versioning/distro', () => {
     expect(di.isEolLts(version)).toBe(expected);
   });
 
-  it('retrieves most recent release schedule with version', () => {
+  it.each`
+    version      | expected
+    ${'focal'}   | ${true}
+    ${'groovy'}  | ${true}
+    ${'hirsute'} | ${true}
+    ${'impish'}  | ${true}
+    ${'jammy'}   | ${false}
+    ${'20.04'}   | ${true}
+    ${'20.10'}   | ${true}
+    ${'21.04'}   | ${true}
+    ${'21.10'}   | ${true}
+    ${'22.04'}   | ${false}
+    ${'24.04'}   | ${false}
+  `('isReleased("$version") === $expected', ({ version, expected }) => {
+    expect(di.isReleased(version)).toBe(expected);
+  });
+
+  it('retrieves schedule of the most recent release', () => {
     expect(di.getNLatest(0)).toEqual({
-      codename: 'Jammy Jellyfish',
-      created: '2021-10-14',
-      eol: '2027-04-21',
-      eol_esm: '2032-04-21',
-      eol_server: '2027-04-21',
-      release: '2022-04-21',
-      series: 'jammy',
-      version: '22.04',
+      codename: 'Impish Indri',
+      series: 'impish',
+      created: '2021-04-22',
+      release: '2021-10-14',
+      eol: '2022-07-14',
+      version: '21.10',
```

Comment:
I think you can use `toMatchObject` heere and ony match on `series` and `version`

Reply:
didnt know that, this is much better to work with. thanks :)